### DISCUSSION TITLE
The current state of mucke

### DISCUSSION THREAD

**Discussion by moritz-weber** [moritz-weber/mucke#174] [Announcements]
First of all, thank you for the kind feedback. The issues section of this project is full of great suggestions, and even the bug reports are nice. Some of those bugs look really annoying, and I can imagine how frustrating it must be to not get a reply to those reports. I'm sorry for neglecting mucke and your issues.

That said, mucke has always been a hobby project for me. It was born out of the frustration with existing music players and the boredom of the Covid lockdowns 😉 I enjoy programming, and this is just a creative outlet to do some UI stuff and use a language that I don't use professionally. However, solving some of the more difficult problems can take a lot of energy and thought, and I currently need this for my day job -- probably until the end of the year.

So, I won't be able to do any substantial work on this for the next months. If you want to take things in your own hands, there are basically two options:

1. Fork the project: I'm completely fine with that as long as the name is changed and GPLv3 is kept.
2. Submit PRs: I'll try to merge incoming PRs from time to time. No promises here, though 🙈

Cheers,
Moritz

**Comment by ildar** [comment]
Moritz,
I share the frustration that you're writing about so I thank you for this
player (that I use daily) wholeheartedly.
Thank you and I hope you'll still keep it alive (at least).

**Comment by Serial-ATA** [comment]
Thanks for all your work on the app!

The timing of this is crazy, I was just looking at this last night wondering if it was abandoned :smile:. You've left mucke in a pretty good state, I've been able to use it daily for the last 9 months or so. That's a lot longer than other Android local file players which are typically abandoned in a buggy state. If you aren't able to return to this in the future, I may be able to pick it up. I'd love to keep using it into the future.

**Comment by CentreMetre** [comment]
After I stopped using and paying for spotify and bought all the music i listen too (luckily just one band so it wasnt too much money haha) i found this music player and its great. Its simple, easy to use, and i can play music in just 1 (shuffle all button) to 4 (playing a specific album). Its kinda hard to find just simple software to use so im very appreciative and lucky to of found this app. I even use it on my PC via scrcpy, especially since it keeps stats of my listens which i like. Hopefully youre able to come back and keep it alive if no one else does.

**Comment by AntsyBoi** [comment]
Is mucke back? There have been a few recent commits. Not trying to force any big statement or commitment to maintaining it again I'm just curious about what's going on with Mucke due to activity after 3 years of inactivity.

**Reply by moritz-weber** [reply]
Kinda, I found some free time lately and got to migrate most of the dependencies to a more recent state. Copilot does make this kind of work a lot easier, since I don't have to research every bit of Android plumbing.

So far, I got a rid of a few bugs on more recent Android versions and have basic Android Auto support. I won't say that it's back to early days' level of commitment, but I hope to keep this project alive. Especially since Spotify is getting worse all the time :sweat_smile:

**Comment by moritz-weber** [comment]
mucke is kinda, almost back... The current builds are pretty close to being ready for a new release. Basically all dependencies are updated. Basic Android Auto support is the only new feature so far. For a new release, I just want to add easy log file sharing and a couple of small improvements to Android Auto. If you like to try it, you can find the current builds in the build job artifacts, e.g., https://github.com/moritz-weber/mucke/actions/runs/30659441020/artifacts/8804759837

There is no release schedule, but I hope to get a new version our soon :tm: 

PS: make sure to export your data for a backup before trying any experimental version.

**Reply by AntsyBoi** [reply]
This is the only good thing to have happened in 2026

**Reply by jannis701** [reply]
thank you so much! unfortunately the controls in the notification did not work on new android releases, so was not able to play/stop or to rate songs from the locked screen for a long time.... (fortunately) I did not find an other app worthy to replace mucke (and I tried out quite a number of apps), so I sticked to it and I am so happy that I now have the full mucke-functionality back!
